interfaces/acados_template/acados_template/AcadosOcpSolver.py

```diff
@@ -539,7 +539,7 @@ def cost_set(self, stage_, field_, value_):
         return
 
 
-    def constraints_set(self, stage_, field_, value_):
+    def cons_set(self, stage_, field_, value_):
```

Comment:
I think `constraints_set` is more clear and not too long.
If you really want to abbreviate it, I would use `constr` as in the Mex interface.
`cons` could be misinterpreted as constant.

Reply:
Fair enough. I like it better too, now that I think about it. Then let's make the convention that all setters must be <full_module_name>_set.

Reply:
@giaf?